pytest.ini

```diff
@@ -34,11 +34,12 @@ testpaths =
     tests/integration/
 env =
     # a weird timezone to check that tests aren't assuming the local timezone
-    TZ=XXX-09:35
+    # Note: this doesn't work properly with pytest-xdist
+    # TZ=XXX-09:35
```

Comment:
This prevents the new test from failing.

Setting the TZ variable in this way doesn't work as intended.

A solution will arrive in https://github.com/cylc/cylc-flow/pull/5826